client/main.lua

```diff
@@ -49,15 +43,48 @@ local function removeElements()
     currentZones = {}
 end
 
+local function getPaid()
+    TriggerServerEvent("qbx_truckerjob:server:getPaid")
```

Comment:
```suggestion
    TriggerServerEvent('qbx_truckerjob:server:getPaid')
```
Change this and the other double quotes to single quotes.

Reply:
In a separate PR to make things clear (these are old lines of code, just moved)